src/com/layout/col/col.js

```diff
@@ -0,0 +1,30 @@
+import {h, Component} from 'preact/preact';
+
+export default class Col extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render({children, flex = 1, flexGrow, flexShrink, flexBasis = 100}) {
+    // let props = Object.assign({}, this.props, {
+    //   flex: undefined,
+    //   flexBasis: undefined,
+    //   flexGrow: undefined,
+    //   flexShrink: undefined
+    // });
+
+    flexGrow = flexGrow != undefined
```

Comment:
To be clear, this test will pass if flexGrow is `undefined` or `null`. In this case, checking for `null` is preferred. 

If you explicitly want to test for `undefined` versus "not set", use `!==`. If it's inconsequential, a `!=` check is technically faster.

Reply:
This block of code is short and simple. Prefer a single line.

```js
flexGrow = (flexGrow != null) ? flexGrow : flex;
```